setup.cfg

```diff
@@ -0,0 +1,24 @@
+[bumpversion]
+current_version = "4.3.0.dev"
+commit = True
+tag = True
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<release>[a-z]+))?
+serialize =
+	{major}.{minor}.{patch}.{release}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
+optional_value = prod
+first_value = dev
+values =
+	dev
+	prod
+
+[bumpversion:file:setup.py]
+search = version = "{current_version}"
+replace = version = "{new_version}"
```

Comment:
where does it get the `new_version` from? if we change `setup.cfg` `[bumpversion]` `current_version` from `4.3.0.dev` to `4.3.0.dev2` for example

it's nice to not have to update the version number in 2 places, but we should make sure we are not adding more complexity to remember steps of the "right" way to update version and run `bump2version`. (possible alternative with less steps to remember: CI could check version number in 2 places are the same)

Reply:
`new_version` is either:

-  from autoincrementing `current_version`  when used with 'bump2version minor' for example.
- specified at the cli with '--new-version 0.6.1' for example.

General, you don't change `current_version` manually, bumpversion itself will increment it to what `new_version` is.
`current_version` is used to build the patterns to find where the `new_version` will be substituted.

Conceptually, `make dev/bumpversion-patch` etc is the 'right' way to bump the version[1]. It then creates `new_version` based on `current_version` and updates all the places where the version needs to be specified.

[1] Or manually running 'bump2version'